server/views/review/includes/bassRequest.pug

```diff
@@ -0,0 +1,15 @@
+include ./itemAndError
+-var request = data.bassReferral && data.bassReferral.bassRequest || {}
+-var errors = errorObject.bassReferral && errorObject.bassReferral.bassRequest || {}
+
+div.pure-g.borderBottomLight.midPaddingTopBottom
+    div.pure-u-1-2
```

Comment:
This means that on mobile it will be half screen too. Is this what you want? usually you would need .pure-u-1.pure-u-md-1-2 so that it is full width on mob and half on bigger screens

Reply:
done